src/components/Table/components/RenderRowOrNestedRow/components/ContentCell/ContentCell.tsx

```diff
@@ -37,38 +37,29 @@ const ContentCell: React.FC<Props> = ({
   dataTestIdPrefix,
   rowIndex,
   index,
-}) => {
-  const theme = useTheme();
-
-  return (
-    <TableCell
-      textAlign={align ? align : columnsHasNumberArr[cellCounter] ? 'right' : 'left'}
-      colSpan={colSpan}
-      type={cellType}
-      padded={padded}
-      width={columnsWithWidth[cellCounter] ? `${columnsWithWidth[cellCounter]}%` : 'initial'}
-      dataTestIdPrefix={dataTestIdPrefix}
-      rowIndex={rowIndex}
-      index={index}
-    >
-      {rowType === 'nested-header' && (
-        <div
-          css={{
-            color: theme.utils.getColor('lightGray', 700),
-            fontSize: theme.typography.fontSizes['14'],
-          }}
-        >
-          {/* nested header render */}
-          {columns[cellCounter]}
-        </div>
-      )}
-      {isComponentFunctionType(content) ? (
-        content({ content, colSpan })
-      ) : (
-        <span data-column={columns[cellCounter]}>{content}</span>
-      )}
-    </TableCell>
-  );
-};
+}) => (
+  <TableCell
+    textAlign={align ? align : columnsHasNumberArr[cellCounter] ? 'right' : 'left'}
```

Comment:
as you touched this part can we give `columnsHasNumberArr[cellCounter]` this a const name and use this. It will be readable